components/feature/contextmenu/src/main/java/mozilla/components/feature/contextmenu/ContextMenuCandidate.kt

```diff
@@ -427,14 +427,17 @@ private fun HitResult.isImage(): Boolean =
 private fun HitResult.isVideoAudio(): Boolean =
     (this is HitResult.VIDEO || this is HitResult.AUDIO) && src.isNotEmpty()
 
-private fun HitResult.isLink(): Boolean =
+private fun HitResult.isHttpLink(): Boolean =
     ((this is HitResult.UNKNOWN && src.isNotEmpty()) || this is HitResult.IMAGE_SRC) &&
         getLink().startsWith("http")
 
+private fun HitResult.isLink(): Boolean =
```

Comment:
```suggestion
private fun HitResult.isUri(): Boolean =
```

isURI would probably be more accurate of a name.

Reply:
I'll update thanks